samples/azure/azure-service-fabric-routing/sample.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Azure Service Fabric Partition Aware Routing
+reviewed: 2017-02-14
+component: Core
+related:
+- nservicebus/azure-service-bus
+---
+
+
+## Prerequisites
```

Comment:
@DavidBoike I've added an introductory paragraph stating the overall purpose of the sample. I've also added `summary` metadata tag to enable better search results.